src/WinUI/ArcGIS.WinUI.Viewer/Samples/GraphicsOverlay/CreateAndEditGeometries/readme.md

```diff
@@ -0,0 +1,51 @@
+
+Use the control panel to undo or redo changes made to the geometry, delete a selected element, save the geometry, stop the editing session and discard any edits, and remove all geometries from the map.
+
+## How it works
+
+1. Create a `GeometryEditor` and set it to the MapView using `MyMapView.GeometryEditor`.
+2. Start the `GeometryEditor` using `GeometryEditor.Start(GeometryType)` to create a new geometry or `GeometryEditor.Start(Geometry)` to edit an existing geometry.
+    * If using the Geometry Editor to edit an existing geometry, the geometry must be retrieved from the graphics overlay being used to visualize the geometry prior to calling the start method. To do this:
+        * Use `MapView.IdentifyGraphicsOverlayAsync(...)` to identify graphics at the location of a tap.
+        * Access the `MapView.IdentifyGraphicsOverlayAsync(...)`.
+        * Find the desired graphic in the `results.FirstOrDefault()` list.
+        * Access the geometry associated with the `Graphic` using `Graphic.Geometry` - this will be used in the `GeometryEditor.Start(Geometry)` method.
+
+3. Create `VertexTool` or `FreehandTool` objects which define how the user interacts with the view to create or edit geometries, setting `GeometryEditor.Tool`.
+4. Check to see if undo and redo are possible during an editing session using `GeometryEditor.CanUndo` and `GeometryEditor.CanRedo`. If it's possible, use `GeometryEditor.Undo()` and `GeometryEditor.Redo()`.
+5. Check whether the currently selected `GeometryEditorElement` can be deleted (`GeometryEditor.SelectedElement.CanDelete`). If the element can be deleted, delete using `GeometryEditor.DeleteSelectedElement()`.
+6. Call `GeometryEditor.Stop()` to finish the editing session and store the `Graphic`. The `GeometryEditor` does not automatically handle the visualization of a geometry output from an editing session. This must be done manually by propagating the geometry returned into a `Graphic` and adding it to the `GraphicsOverlay`.
+    * To create a new `Graphic` in the `GraphicsOverlay`:
+        * Using `Graphic(Geometry)`, create a new Graphic with the geometry returned by the `GeometryEditor.Stop()` method.
+        * Append the `Graphic` to the `GraphicsOverlay`(i.e. `GraphicsOverlay.Graphics.Add(Graphic)`).
+    * To update the geometry underlying an existing `Graphic` in the `GraphicsOverlay`:
+        * Replace the existing `Graphic`'s `Geometry` property with the geometry returned by the `GeometryEditor.Stop()` method.
+
+## Relevant API
+
+* Geometry
+* GeometryEditor
+* Graphic
+* GraphicsOverlay
+* MapView
+
+## Additional information
+
+The sample opens with the ArcGIS Imagery basemap centered on the island of Inis Me�in (Aran Islands) in Ireland. Inis Me�in comprises a landscape of interlinked stone walls, roads, buildings, archaeological sites, and geological features, producing complex geometrical relationships.
```

Comment:
The `á` character is not rendering properly in the sample viewer.

Reply:
Maybe we change the text to Inishmaan unless you know of a workaround?

Reply:
I think for this case just use `a` rather than `á` it's close enough for our purposes.